lib/ansible/plugins/callback/__init__.py

```diff
@@ -555,11 +555,32 @@ def v2_runner_on_skipped(self, result: TaskResult) -> None:
             host = result._host.get_name()
             self.runner_on_skipped(host, self._get_item_label(getattr(result._result, 'results', {})))
 
-    def v2_runner_on_unreachable(self, result):
+    def v2_runner_on_unreachable(self, result: TaskResult) -> None:
+        """Get details about a task that Ansible could not perform because it could not reach a
+        targeted host, then process the details as required by the callback (output, profiling,
+        logging, notifications, etc.)
```

Comment:
https://peps.python.org/pep-0257/ suggests that the summary line sentence should fit in the first line. Most linters also cap that line to 72 chars. And the following description blob is usually allowed to have longer lines.

Reply:
I split the docstring into a summary line and description per PEP 257, and I reduced the size of all but one of the docstrings to less than 72 characters per PEP 8 (the summary line for `v2_runner_on_unreachable` is 73 characters long).

Reply:
I think that's why it's called a summary *line*, not a summary paragraph :man_shrugging:
It's not an explanation but a very concise summary, as short as possible. And so it's typically separate from the explanatory paragraph that follows it.

Reply:
Marking this thread as resolved.